src/gs-shell-overview.c

```diff
@@ -159,7 +159,6 @@ gs_shell_overview_get_popular_cb (GObject *source_object,
 		goto out;
 	}
 	/* Don't show apps from the category that's currently featured as the category of the day */
```

Comment:
Well, one line generated less possible conflicts then 20 but you got a point. Thanks (and thanks for the warning about the forgotten comment...).